### ISSUE TITLE
Adding second game with "add games" crashes the system

### ISSUE THREAD

**Issue by Favna** [Robz8/TWLoader#128]
When selecting a second game when using the "add game" feature the entire TWLNAND just crashes. The error as shown on the bottom screen can be seen in this video:

[video](http://streamable.com/xa8a7)

Like I say in the video, I already have the .ini files which is why you see that in the first game added, but that is not relevant to the issue as it also occurs when it does create an ini file for the game.

I updated my TWLoader on my R4 at flashcardroot/_nds/twloader.nds as according to the release log it was updated

My settings.ini is as follows:

```ini
[FRONTEND]
COLOR = 0
TOP_BORDER = 1
TOP_LAYOUT = 0
COUNTER = 0
AUTOUPDATE = 0
AUTODOWNLOAD = 0
ROM_FOLDER = _nds/twloader/roms
FCROM_FOLDER = _nds/twloader/roms
LANGUAGE = -1
MENU_COLOR = 0
SHOW_FILENAME = 0
CUSTOM_BOTTOM = 0
NAME = Favna

[TWL-MODE]
RAINBOW_LED = 0
TWL_CLOCK = 1
BOOT_ANIMATION = 1
HEALTH&SAFETY_MSG = 1
LAUNCH_SLOT1 = 1
RESET_SLOT1 = 1
DEBUG = -1
FORWARDER = 1
FLASHCARD = 2
TWL_VRAM = 0
LOCK_ARM9_SCFG_EXT = 0
SLOT1_KEEPSD = 0
GBARUNNER = 0
GAMESETTINGS_PATH = sd:/_nds/twloader/gamesettings/flashcard/_nds/twloader.nds
```

Due to length I have put my log on [hastebin, see here](https://hastebin.com/hitasotuze.xml)

Screenshots of files on mSD card (3ds):

![image](https://cloud.githubusercontent.com/assets/4019718/22396314/cbe2f75e-e555-11e6-804d-ddf50867ce3c.png)

![image](https://cloud.githubusercontent.com/assets/4019718/22396316/cfe44d58-e555-11e6-8e5f-8ebf8ee8c1b9.png)

Also including gamesettings because it is referenced in settings but i've never seen any files being created there 

![image](https://cloud.githubusercontent.com/assets/4019718/22396319/df62bac6-e555-11e6-93cc-5855fcf545ab.png)

I updated to 3.1 by installing the .cia for the side and gui apps as well as removing the entire _nds folder from my 3DS sd and Flashcard sd then replacing them with the newly downloaded ones. Proceeded to put banner image files back in the appropriate folder for 3DS but rest and flashcard untouched.

This issue has also been noticed by @Margen67

**Comment by Jolty95** [comment]
Shall I close this too?

**Comment by Favna** [comment]
No this still occurs even with the latest beta build and since it's in the "add games" part I think it's more related to the TWLNAND-side rather than the gui

**Comment by Jolty95** [comment]
Solved in v3.2 ;)
